src/Compilers/CSharp/Portable/Symbols/ReferenceManager.cs

```diff
@@ -415,9 +412,12 @@ private bool CreateAndSetSourceAssemblyFullBind(CSharpCompilation compilation)
                 }
 
                 // Setup references for the compilation (out parameters)
-                referencedAssembliesMap = new Dictionary<MetadataReference, ReferencedAssembly>(referenceMap.Length);
+                referencedAssembliesMap = new Dictionary<MetadataReference, int>(referenceMap.Length);
                 referencedModulesMap = new Dictionary<MetadataReference, int>(modules.Length);
 
+                var referencedAssemblies = ArrayBuilder<AssemblySymbol>.GetInstance();
+                var aliasesOfReferencedAssemblies = ArrayBuilder<ImmutableArray<string>>.GetInstance();
+
                 var sourceModule = assemblySymbol.SourceModule;
                 var referencedAssemblySymbols = sourceModule.GetReferencedAssemblySymbols();
```

Comment:
It looks like this local is no longer used. Do we really have to rebuild this list in referencedAssemblies builder and allocate new array?